qutebrowser/browser/signalfilter.py

```diff
@@ -56,9 +56,10 @@ def create(self, signal, tab):
         Return:
             A partial function calling _filter_signals with a signal.
         """
-        return functools.partial(self._filter_signals, signal, tab)
+        log_signal = debug.signal_name(signal) not in self.BLACKLIST
+        return functools.partial(self._filter_signals, signal, log_signal, tab)
 
-    def _filter_signals(self, signal, tab, *args):
+    def _filter_signals(self, signal, log_signal, tab, *args):
```

Comment:
This is getting kinda error-prone - what about making `signal`, `log_signal` and `tab` keyword-only arguments?